bin/check_formatting.sh

```diff
@@ -31,7 +31,10 @@ done
 echo "Checking for trailing whitespace"
 # git grep returns a 0 status if there is a match
 # so we negate the result for consistency
-! git grep --line-number ' $'
+# binaries and files directly copied from exercism/problem-specifications are ignored
+! git grep -I --line-number ' $' -- . \
```

Comment:
The `-I` flag ignores binary files, because without it the check always fails for `exercises/concept/file-sniffer/assets/elf.o` locally. I'm not sure why it doesn't fail on CI, maybe a different version of git, but it doesn't really make sense to format binaries anyway.